packages/lodestar-utils/src/json.ts

```diff
@@ -15,11 +15,10 @@ export function toJson(arg: unknown): Json {
 
     case "object":
       if (arg === null) return "null";
-      if (Array.isArray(arg)) return arg.map(toJson);
       if (arg instanceof Uint8Array) return toHexString(arg);
       if (arg instanceof LodestarError) return toJson(arg.toObject());
       if (arg instanceof Error) return toJson(errorToObject(arg));
```

Comment:
I think only circular references are here and this is totally wrong since external errors have huge metadata. 600+lines so in #1665 I've changed this to just print error.message.

Reply:
Oh damn, then I'll pick only message and stack `errorToObject`. Would that resolve your point?

Reply:
Probably, yeah